PYME/recipes/pointcloud.py

```diff
@@ -381,3 +384,101 @@ def execute(self, namespace):
             pass
         
         namespace[self.outputName] = res
+
+
+@register_module('BICGaussianMixtureModel')
+class BICGaussianMixtureModel(ModuleBase):
```

Comment:
worth considering a refactor to `GaussianMixtureModel` with 'BIC selection of `n_components` as an option (could be an enum, with 2 options to start with, BIC and manual, which would let us expand to other selection methods later).